common/meta/raft-store/src/state_machine/sm.rs

```diff
@@ -314,14 +316,17 @@ impl StateMachine {
                 }
             }
 
-            Cmd::CreateDatabase { ref name } => {
+            Cmd::CreateDatabase {
+                ref name,
+                ref engine,
```

Comment:
It'd better have database information, in our case, the `engine`  in a `DatabaseMeta`.

Reply:
got it.